packages/debugger/.env.sample

```diff
@@ -3,4 +3,6 @@
 FARCASTER_DEVELOPER_MNEMONIC=
 # (OPTIONAL) Only needed for the debugger to create a real Farcaster signer. Get this by visiting your Warpccast profile, pressing the kebab (three dots) menu and then "About" and then your fid should be there.
 # Example: FARCASTER_DEVELOPER_FID=1214
-FARCASTER_DEVELOPER_FID=
\ No newline at end of file
+FARCASTER_DEVELOPER_FID=
+
+NEXT_PUBLIC_WALLETCONNECT_ID=
```

Comment:
this is necessary only to build the debugger? What happens when we build the debugger and then publish it to npm? Is that kept in the package or do developers need to provide their own value?